Source/PeleC.cpp

```diff
@@ -152,6 +155,10 @@ bool         PeleC::do_mol_load_balance = false;
 SprayParticleContainer* PeleC::SprayPC = nullptr;
 #endif
 
+#ifdef SOOT_MODEL
+SootModel* PeleC::soot_model = new SootModel();
```

Comment:
is there a way we can avoid raw pointers? `unique_ptr` or whatnot and then avoid all the `delete` and stuff like that?

Reply:
Yep. I will update that.